### PR TITLE
Doc/darklight theme

### PR DESCRIPTION
**Pull request by Moneyba**
## Changes
- Add [darklight theme](https://docsify-darklight-theme.boopathikumar.me/#/)
- Add footer

## Checks
- [ ] Project Builds
- [ ] Project passes tests and checks
- [ ] Updated documentation accordingly

### CODE REVIEW

File: docs/custom.css [hunk 1]
```diff
@@ -1,18 +1,37 @@
...
+    background-repeat: no-repeat;
+    background-image: var(--toogleImage);
+    background-size: cover;
+    cursor: pointer;
+    transition: background-image .15s ease-in-out .15s
+}
+
+#docsify-darklight-theme p {
+    visibility: hidden
+}
```
Comment (AdrianLorenzoDev) [inline docs/custom.css:16]: Wouldn't this make this theme to hide the paragraphs?

**Review by AdrianLorenzoDev — APPROVED** [review]
LGTM!

File: docs/custom.css [hunk 2]
```diff
@@ -1,18 +1,37 @@
...
+    --code-inline-background: var(--code-theme-background) !important;
     --sidebar-toggle-icon-color: var(--theme-color);
     --navbar-root-color--hover: var(--theme-color);
+    --content-max-width: 90% !important;
+    --link-text-decoration: none !important;
 }
 
 .sidebar > h1 img {
-    max-width: 60%;
+    max-width: 55% !important;
```
Comment (davidverdu) [inline docs/custom.css:30]: Can we avoid the usage of !important some way?

File: docs/index.html [hunk 3]
```diff
@@ -20,14 +22,12 @@
   </style>
 </head>
 <body>
-
-
 <div id="app"></div>
 <script>
   window.$docsify = {
     homepage: 'chapters/01_introduction.md',
-    logo: 'img/booster-logo.png',
```
Comment (davidverdu) [inline docs/index.html:29]: In case we have removed the logo image, we should delete the png file too. Don't you think?

File: docs/index.html [hunk 4]
```diff
@@ -53,13 +53,20 @@
       function(hook, vm) {
         hook.beforeEach(function (html) {
           const file = vm.router.getFile().replace('/booster/','');
-          let url = 'https://github.com/boostercloud/booster/tree/main/docs/' + file;
```
Comment (davidverdu) [inline docs/index.html:56]: why let is not suitable here if this is a variable that can change its value?

**Review by davidverdu — APPROVED** [review]
Visually it looks good to me. Related the code I added some comments. Good job!

File: docs/index.html (hunk 3 above)
Comment (Moneyba) [reply docs/index.html:29]: The logo is used by the Readme.md file.

Comment (davidverdu) [reply docs/index.html:29]: Beware the fact that the logo png could be used in the Github Readme or contrib pages when you see them from Github repository

**Pull request merged**